tutor/env.py

```diff
@@ -77,20 +77,15 @@ def __init__(self, config: t.Optional[Config] = None):
         self.environment = JinjaEnvironment(self.template_roots)
 
         # Filters
-        plugin_filters: t.Iterator[
-            t.Tuple[str, JinjaFilter]
-        ] = hooks.Filters.ENV_TEMPLATE_FILTERS.iterate()
+        plugin_filters = hooks.Filters.ENV_TEMPLATE_FILTERS.iterate()
         for name, func in plugin_filters:
             if name in self.environment.filters:
                 fmt.echo_alert(f"Found conflicting template filters named '{name}'")
             self.environment.filters[name] = func
         self.environment.filters["walk_templates"] = self.walk_templates
 
         # Globals
-        plugin_globals: t.Iterator[
-            t.Tuple[str, JinjaFilter]
-        ] = hooks.Filters.ENV_TEMPLATE_VARIABLES.iterate()
-        for name, value in plugin_globals:
+        for name, value in hooks.Filters.ENV_TEMPLATE_VARIABLES.iterate():
```

Comment:
Nice to see how you were able to simplify the filter usages now that the declarations are typed.